packages/plugin-e2e/tests/as-admin-user/panel/panelEdit.spec.ts

```diff
@@ -0,0 +1,139 @@
+import { test, expect } from '../../../src';
+
+test('selecting value in radio button group', async ({ gotoPanelEditPage }) => {
+  const panelEdit = await gotoPanelEditPage({ dashboard: { uid: 'mxb-Jv4Vk' }, id: '5' });
+  const clockOptions = panelEdit.getCustomOptions('Clock');
+  const countdownOptions = panelEdit.getCustomOptions('Countdown');
+  const clockMode = clockOptions.getRadio('Mode');
+
+  await expect(countdownOptions.element).not.toBeVisible();
+
+  await clockMode.check('Countdown');
+  await expect(clockMode).toHaveChecked('Countdown');
+
+  await expect(countdownOptions.element).toBeVisible();
+});
+
+test('re-selecting value in radio button group', async ({ gotoPanelEditPage }) => {
+  const panelEdit = await gotoPanelEditPage({ dashboard: { uid: 'mxb-Jv4Vk' }, id: '5' });
+  const clockOptions = panelEdit.getCustomOptions('Clock');
+  const countdownOptions = panelEdit.getCustomOptions('Countdown');
+  const clockMode = clockOptions.getRadio('Mode');
+
+  await expect(countdownOptions.element).not.toBeVisible();
+
+  await clockMode.check('Countdown');
+  await expect(clockMode).toHaveChecked('Countdown');
+
+  await expect(countdownOptions.element).toBeVisible();
+
+  await clockMode.check('Time');
+  await expect(clockMode).toHaveChecked('Time');
+});
+
+test('checking switch', async ({ gotoPanelEditPage }) => {
+  const panelEdit = await gotoPanelEditPage({ dashboard: { uid: 'mxb-Jv4Vk' }, id: '5' });
+  const clockOptions = panelEdit.getCustomOptions('Clock');
+  const monospaceFont = await clockOptions.getSwitch('Font monospace');
+
+  await expect(monospaceFont).toBeSwitched({ on: false });
```

Comment:
I think I rather use the standard API: 
```await expect(monospaceFont.locator).not.toBeChecked()```

Reply:
Alternatively, we keep the custom matcher but use the same name and signature as the Playwright one:
```await expect(monospaceFont).not.toBeChecked()```

Reply:
Yeah, I started with that but ran into problems with the standard method "toBeChecked" being overwritten.